.golangci.yml

```diff
@@ -1,45 +1,63 @@
+version: "2"
```

Comment:
I only ran the [golangci migrate command](https://golangci-lint.run/product/migration-guide/), then reapply the timeout and the big comment. But that's it.